src/hotel-management-app/src/app/customer/cart/cart.component.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
Could you add a option to go back in this cart

Reply:
resolved